_posts/2015-02-23-taskcluster-update.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "Taskcluster Release Part 1 : Gecko"
+description: "TC Update 1"
+category: mozilla, ci
+tags: [ci, docker, mozilla]
+---
+
+It's been awhile since my last blog post about taskcluster and I wanted to give an update...
+
+## Taskcluster + Gecko
+
+Taskcluster is running by default on
+
+  - [try](https://treeherder.allizom.org/#/jobs?repo=try)
+  - [b2g-inbound](https://treeherder.allizom.org/#/jobs?repo=b2g-inbound)
+  - [mozilla-inbound](https://treeherder.allizom.org/#/jobs?repo=mozilla-inbound)
+  - [fx-team](https://treeherder.allizom.org/#/jobs?repo=fx-team)
+  - [mozilla-central](https://treeherder.allizom.org/#/jobs?repo=mozilla-central)
+
+
+You will see both buildbot run and taskcluster run jobs the "TC" jobs are
+prefixed accordingly so you can tell the difference.
+
+This is the last big step to enabling TC as the default CI for many mozilla
+projects also new branches are a simple config away.
```

Comment:
I would leave out the 'Also' and say that 'Adding new and existing branches is easily achieved with basic config changes.'